OSTranslatorII/os_translator_ii_dialog.py

```diff
@@ -9,22 +9,29 @@
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
 
-from PyQt4 import QtCore, QtGui
-import traceback
+from __future__ import absolute_import
 import string, multiprocessing
 import xml.etree.ElementTree as ET
 
-from import_manager import *
-from result_dialog import *
-from post_processor_thread import *
-from utils import *
+from .import_manager import *
+from .result_dialog import *
+from .post_processor_thread import *
+from .utils import *
 
-from qgis.core import QgsDataSourceURI
+try:
+    from qgis.core import QgsDataSourceURI
+except ImportError:
+    from qgis.core import QgsDataSourceUri
+
+try:
```

Comment:
This import checks can be done just once in single try ... except ImportError block.